### DISCUSSION TITLE
Composite model not able to get updated column

### DISCUSSION THREAD

**Discussion by VinothiniRaj** [TabularEditor/TabularEditor3#1207] [🙋♀️TE3 Q&A] [answered]
### Description

Hi Team

In my composite model, I have connected these two data models. 

<img width="949" alt="tmp_9b455855-bdd6-4f1e-b389-a4b6f83e61b0" src="https://github.com/TabularEditor/TabularEditor3/assets/153912235/d396a4e3-5013-4a27-8e4e-5a198d73b93b">
<img width="956" alt="tmp_92c36f25-1ed3-4737-abf5-6032a6d48db4" src="https://github.com/TabularEditor/TabularEditor3/assets/153912235/84d8f31a-eb09-4743-8983-9d9bc7d0d7c7">

I have added EntitycodeID column in the first model. 
But am not able to get this column in composite model. 
<img width="170" alt="tmp_b8b195ae-9dc2-4cd8-bcda-fdbec5ab3016" src="https://github.com/TabularEditor/TabularEditor3/assets/153912235/0a45f32e-7242-4dc1-b625-dc6b5d8b190c">
<img width="682" alt="tmp_303ab908-e994-4575-9d18-1a6955e649db" src="https://github.com/TabularEditor/TabularEditor3/assets/153912235/57bde87f-07dc-4ed2-bc8b-f06e3f0e2141">

### Tabular Editor 3 Version

3.12.1

### Windows Version

Windows 10 Pro

**Comment by otykier** [comment, marked as answer]
Hi @VinothiniRaj 
Unfortunately, we do not yet have full support for DQ over AS models in Tabular Editor. For this reason, you will see that the "Update table schema" feature won't work. We're working on adding full support for this, and will provide you with an update once we have something to share.

In the mean time, you will have to work around this limitation by manually adding the column on the DQ over AS model, i.e. add a Data Column to the table, then fill out the "SourceColumnName" property to match the "Name" property of the source column, and set the "SourceLineageTag" property to match the "LineageTag" of the source column.

I also wrote an [article with a C# script](https://www.linkedin.com/pulse/composite-models-tabular-editor-daniel-otykier/?trackingId=McY7Be3UQyyR4jWu%2B%2FdiJg%3D%3D) a while ago to help automate the process of importing tables from a remote model into a DQ over AS composite model. This may also help you in this case, as you could run the script to temporarily import a full copy of the tables that are missing columns, then copy over the columns that you need into the existing tables, and finally delete the temporary tables (created by the script).
